divi/src/miner.cpp

```diff
@@ -28,10 +29,8 @@
 
 extern Settings& settings;
 extern CCoinsViewCache* pcoinsTip;
```

Comment:
This one could be encapsulated as well no?

Reply:
Good catch, not sure why this one and a few others were missed.  I've now done another round of grepping all the code to see if any of the four globals are left.